pkgs/tools/networking/junkie/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, pkgconfig, libpcap, guile, openssl }:
+
+stdenv.mkDerivation rec {
+  name = "junkie-${version}";
+  version = "2.8.0";
+
+  src = fetchurl {
+    url = https://github.com/rixed/junkie/archive/v2.8.0.tar.gz;
+    sha256 = "c9b57bd6e06d4f90e6a88b775f33fa00f66313ba0f663df70198eddf1d4be298";
+  };
+  buildInputs = [ pkgconfig libpcap guile openssl ];
```

Comment:
`pkgconfig` should be in `nativeBuildInputs` because it's only used at build time